### DISCUSSION TITLE
Soft delete in Tags interface

### DISCUSSION THREAD

**Discussion by d1rOn** [directus/directus#16568] [Feature Requests]
I want to suggest to improve Tags interface.

At this time it has 2 problems:
1. The user don't know what will be heppened if he will click on the tag.
2. Intentional or accidental pressing provokes the removal of the tag.

I suggest:
1. When hovering over a tag, change the cursor to a pointer and display a delete-icon.
2. When deleting a tag, do not delete it completely, but display it as strikethrough or translucent. Thus, when pressed again, the user will be able to return it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
